zone.go

```diff
@@ -268,6 +268,19 @@ type newZone struct {
 	Account *Account `json:"organization,omitempty"`
 }
 
+// FallbackOrigin describes a fallback origin
+type FallbackOrigin struct {
+	Value    string `json:"value"`
+	ID       string `json:"id,omitempty"`
+	Editable bool   `json:"editable,omitempty"`
```

Comment:
The `Editable` field isn't something we really need here. Can we do without it?

Reply:
I mainly left this in as I saw this field had been kept for other configurations (e.g. `ZoneSSLSetting`), but happy to remove.